src/main/java/com/jvm_bloggers/core/newsletter_issues/domain/NewsletterIssue.java

```diff
@@ -58,6 +61,7 @@
     @Singular
     @NonNull
     @OneToMany(fetch = FetchType.EAGER)
+    @Fetch(value = FetchMode.SUBSELECT)
```

Comment:
Yes, that was my first solution to fix this. However, replacing `List` with `Set` caused the change being propagated up into the business logic so there were a lot of changes in many files. I don't know if we depend on `List` semantics either so changing to `Set` has some risk. Eventually I decided to apply a less invasive approach - a single annotation.